src/shared/js/components/Sidekick/AddNewPageDialog.js

```diff
@@ -13,66 +13,61 @@ import {
+  //   // const pageParams = new URLSearchParams(window.location.search);
+  //   // setPageId(pageParams.get('id'));
 
-  const [loading, setLoading] = React.useState(false);
-  const [message, setMessage] = React.useState();
-  const Contentful = useAuth();
-  const [pageId, setPageId] = useState('');
-  useEffect(() => {
-    // TODO read this from a meta tag. This assumes on preview route
-    const pageParams = new URLSearchParams(window.location.search);
-    setPageId(pageParams.get('id'));
+  //   async function fetchTemplates() {
+  //     try {
+  //       const response = await client.getEntries({
+  //         content_type: 'template'
+  //       });
+  //       // console.log('templates', response.items);
+  //       // setTemplates(response.items);
 
-    async function fetchTemplates() {
-      try {
-        const response = await getTemplates();
-        console.log('templates', response.items);
-        setTemplates(response.items);
-      } catch (err) {
-        console.log('error', err);
-      }
-    }
-    fetchTemplates();
-  }, []);
-  const groupedTemplates = templates.reduce((acc, template) => {
-    if (!template.fields.category) return acc;
-    if (!acc[template.fields.category.toUpperCase()]) {
-      acc[template.fields.category.toUpperCase()] = [];
-    }
-    acc[template.fields.category.toUpperCase()].push(template);
-    return acc;
-  }, {});
-  const handleClick = (template) => async () => {
-    try {
-      setMessage();
-      setLoading(true);
-      await insertTemplateOnPage(pageId, template.sys.id, index);
-      setTimeout(() => {
-        window.postMessage({ type: 'REFRESH_CONTENT' }, '*');
-      }, 500);
+  //     } catch (err) {
+  //       console.log('error', err);
+  //     } finally {
+  //       setLoading(false);
+  //     }
+  //   }
+  //   fetchTemplates();
+  // }, [client]);
+  // const groupedTemplates = templates.reduce((acc, template) => {
+  //   if (!template.fields.category) return acc;
+  //   if (!acc[template.fields.category.toUpperCase()]) {
+  //     acc[template.fields.category.toUpperCase()] = [];
+  //   }
+  //   acc[template.fields.category.toUpperCase()].push(template);
+  //   return acc;
+  // }, {});
+  // const handleClick = (template) => async () => {
+  //   try {
+  //     setMessage();
+  //     setLoading(true);
+  //     await insertTemplateOnPage(pageId, template.sys.id, index);
+  //     setTimeout(() => {
+  //       window.postMessage({ type: 'REFRESH_CONTENT' }, '*');
+  //     }, 500);
 
-      handleClose();
-      setMessage('Template inserted in ' + pageId);
-    } catch (err) {
-      console.log('InsertTemplateError', err);
-      setMessage('Error' + err.message);
-    }
-    setLoading(false);
-  };
+  //     handleClose();
+  //     setMessage('Template inserted in ' + pageId);
+  //   } catch (err) {
+  //     console.log('InsertTemplateError', err);
+  //     setMessage('Error' + err.message);
+  //   }
+  //   setLoading(false);
+  // };
```

Comment:
Let's remove this code given that this feature is not fully implemented and implement it in the future.